app/components/forms/orders/attendee-list.js

```diff
@@ -2,20 +2,24 @@ import Component from '@ember/component';
 import { computed } from '@ember/object';
 import { groupBy } from 'lodash-es';
 
-export default Component.extend({
-  buyer: computed('data.user', function() {
-    return this.get('data.user');
-  }),
-  holders: computed('data.attendees', function() {
-    return this.get('data.attendees');
-  }),
-  isPaidOrder: computed('data', function() {
-    if (!this.get('data.amount')) {
-      return false;
-    }
-    return true;
-  }),
-  allFields: computed('fields', function() {
+export default class extends Component {
+  @computed('data.user')
+  get buyer() {
+    return this.data.user;
+  }
+
+  @computed('data.user')
+  get holders() {
+    return this.data.attendees;
+  }
+
+  @computed('data.amount', 'data.isBillingEnabled')
```

Comment:
use or

Reply:
did it.

Reply:
No, Use like this 
```js
@or('data.amount','data,isBillingEnabled')
showBillingInfo;
``

Reply:
@kushthedude it doesn't work this way.

Reply:
@kushthedude is right. Thats the right way to do it.
If it doesn't work, maybe you're doing something wrong. You can push the code.

Reply:
@niranjan94 Alright, taking a look.